README.md

```diff
@@ -29,13 +29,13 @@ To install packages run
     $ npm install
 
 #### Development mode with different API environments
-To run locally in development mode with the development API run
+To run locally in development mode with the development API and with the regular unencrypted API keys, run:
 
     $ clientId=[client id] clientSecret=[client secret] npm run dev-api-start
 
-To run locally in development mode with the production API run
+To run locally in production mode with the production API and with encrypted API keys, run:
```

Comment:
hahaha, if so, may I suggest that we move the section about `Client Id and Secret` in front of `Development mode with different API environments`? Because here, we are talking about something a reader might have not encountered yet. I think it is a good order to provide basic information and explanation for special terms first and then later tell the reader how to use them.